Source/CarthageKit/Xcode.swift

```diff
@@ -151,6 +151,10 @@ private struct ProjectEnumerationMatch: Comparable {
 				return .success(ProjectEnumerationMatch(locator: .Workspace(URL), level: enumerator.level))
 			} else if (UTTypeConformsTo(typeIdentifier, "com.apple.xcode.project") != 0) {
 				return .success(ProjectEnumerationMatch(locator: .ProjectFile(URL), level: enumerator.level))
+			} else if (UTTypeConformsTo(typeIdentifier, "public.symlink") != 0) {
```

Comment:
Of course, not sure what to return if that call fails, though, currently doing a `.ReadFailed(URL, nil)` - maybe that's good enough, because this never actually fails. The docs only say:

> If some symbolic links cannot be resolved, this property contains those broken symbolic links.

whatever that means  ¯\_(ツ)_/¯